config/logger.js

```diff
@@ -0,0 +1,23 @@
+const { createLogger, format, transports } = require('winston');
+const winston = require('winston');
+
+module.exports = createLogger({
+  level: 'info',
+  format: format.combine(
+    format.simple(),
+    format.timestamp(),
+    format.printf(info => `[${info.timestamp}] ${info.level}: ${info.message}`)
```

Comment:
Add `file name` and `line number` to improve the readability of the log

Reply:
I found file name in "module". But couldn't figure out line number. need help

Reply:
In which which place, I should place logger? in API?